concepts/0302-aries-interop-profile/README.md

```diff
@@ -91,10 +91,69 @@ Feature | [0036-issue-credential](https://github.com/hyperledger/aries-rfcs/tree
 Feature | [0037-present-proof](https://github.com/hyperledger/aries-rfcs/tree/4fae574c03f9f1013db30bf2c0c676b1122f7149/features/0037-present-proof)
 Feature | [0056-service-decorator](https://github.com/hyperledger/aries-rfcs/tree/527849ec3aa2a8fd47a7bb6c57f918ff8bcb5e8c/features/0056-service-decorator)
 
-#### Test Suite
+#### AIP v1.0 Test Suite
 
 > To Do: Link(s) to version(s) of the test suite/test cases applicable to this Aries Interop Profile version.
 
+### Aries Interop Profile Version: 2.0
+
+The following are the goals used in selecting RFC versions for inclusion in AIP 2.0, adn the RFCs added as a result of each goal:
+
+- From AIP 1.0: Aries Agents must be able to establish connections, exchange credentials and complete a connection-less proof-request/proof transaction.
+- Aries agents must be able to reuse connections.
+  - RFCs 0434, 0023, 0519, 0360
```

Comment:
> 0360

I submitted a [proposal to move away from did:key](https://github.com/hyperledger/aries-rfcs/pull/577). A primary motivator there is to achieve better cryptographic agility with the ultimate goal of supporting NIST curves.

Supporting NIST curves is very important to us.

Reply:
Are you going to present on this on Wednesday?  This was up in the air when we adjourned before the holidays and you were investigating.

Reply:
@swcurran I can lead a chat in wednesday's call. @dhh1128 is also interested.